research-hub-web/src/app/components/home/research-activity/research-activity.component.scss

```diff
@@ -1,31 +1,123 @@
-@import '../../../../styles.scss';
+@import "../../../../styles.scss";
+@import '../../../../themes.scss';
 @import '../../../../typography.scss';
+@import '../home.component.scss';
 
 
-/*
- * Sets the height of the research chips on the home page and sets the pointer style. Button size is increased at xs (corresponding to when flex layout is switched to column-based) to improve usability on mobile devices.
+/**
+ * Custom Text
  */
-
-.ra-chip {
-  border-radius: 10vh;
-  $ra-chip-height: 4.625vh;
-  $ra-chip-height-touch: 6vh;
-  $ra-chip-padding: 1.2em;
-  $ra-chip-font-size: 1.8vh;
-
-  cursor: pointer;
-  height: $ra-chip-height;
-  padding-right: $ra-chip-padding;
-  padding-left: $ra-chip-padding;
-  color: white !important;
-  font-size: $ra-chip-font-size!important;
-
-  &.tablet-view {
-    height: $ra-chip-height-touch;
+ .text {
+  width: 100%;
+  @extend .text;
+  @include size(large) { 
+    text-align: left;
   }
+}
+
 
-  &.mobile-view {
-    height: $ra-chip-height-touch;
+/**
+ * Research Activity Container
+ */
+.research-activity-content {
+  & .outer-panel {
     width: 100%;
+    height: 240px;
+    text-align: center;
+    box-shadow: 0px -3px 10px rgba(0, 0, 0, 0.3);
+
+    // For the box-shadow
+    display: table;
+    overflow: hidden;
+
+
+    &:active, &:focus{
+      outline: none;
+      text-decoration: none;
+    }
+    & .inner-panel {
+
+      margin-right: 20px; /* Set to 0 if you don't want shadow at the right side */
+      margin-left: 0px; /* Set to 20px if you want shadow at the left side */
+      margin-top: 0px; /* Set to 20px if you want shadow at the top side */
+      margin-bottom: 0px; /* Set to 20px if you want shadow at the bottom side */
+      box-shadow: -3px 0px 10px black; 
+
+      width: 100%;
+      margin: 0 auto;
+      transform: skew(0deg); 
+      margin-left: 0vw;
+      height: 240px; 
+      display: flex;
+      text-align: center;
+      align-items: center;
+      justify-content: center;
+      background: map-get($hub-teal, 500);
+      border: 4px solid map-get($hub-teal, 500);
+      transition: background 0.11s ease-in-out, color 0.1s ease-in-out;
+      @include size(large) {
+        width: 55%;
+        margin-left: -5vw;
+        transform: skew(30deg);
+      }
+      & * {
```

Comment:
Again here, all of these selectors are redundant.

```scss
.hub {
    .sam { color: #ccc }
}

```
Equals:

```html
<element class="hub">
  …
    <element class="sam">
```
---
And:
```scss
.hub {
    &.sam { color: #ccc }
}
```

Equals: 

```html
<element class="hub sam">
```
---
But what you're doing here:
```scss
.hub {
    & .sam { color: #ccc }
}
```

Equals: 

```html
<element class="hub">
  …
    <element class="sam">
```

I.e. the same thing as the first option, the `&` is not needed. That's the whole point of nesting classes in Sass.